server/graphql/v2/object/Host.js

```diff
@@ -534,9 +549,47 @@ export const Host = new GraphQLObjectType({
             const collectiveIds = collectives.map(collective => collective.id);
             where.CollectiveId = { [Op.in]: collectiveIds };
           }
+
+          const contributionAmountOverTime = async () => {
+            let contributionAmountOverTime;
+            if (args.timeUnit) {
```

Comment:
Can this be removed as args.timeUnit should be non-null?

Reply:
Yes, thanks for pointing it out.